LEAF_Request_Portal/admin/templates/mod_templates_email.tpl

```diff
@@ -240,6 +240,17 @@
     const allowedToCcFormats = {
         "orgchart_employee": 1,
     }
+
+    $(function(){
+        $('#emailToCode').on("keyup", function(e) {
+            this.value = this.value.replace(/[;:\s]/, ',');
```

Comment:
The instructions say to place an address "one per line". Commas should not be used.

Reply:
Modified the code to separate each email with a new line character to match our instructions. Also, made it so that you can't have more than one blank line.